README.md

```diff
@@ -141,34 +141,44 @@ List<Project> allProjects = projectPager.all();
 
 ---
 ## Java 8 Stream Support
-As of GitLab4J-API 4.9.2, you can also stream list based items in a Java 8 Stream using a Pager instance as follows:
+As of GitLab4J-API 4.9.2, all GitLabJ-API methods that return a List result also similarlly named method returns a Java 8 Stream.  The Stream returning methods use the following naming convention:  ```getXxxxxStream()```.  For example:
+
 ```java
-// Get a Pager instance to get a Stream<Project> instance.
-Pager<Project> projectPager = gitlabApi.getProjectsApi().getProjects(10);
+// List method
+List<Project> projects = gitlabApi.getProjectApi().getProjects();
 
-// Stream the Projects printing out the project name.
-projectPager.stream().map(Project::getName).forEach(name -> System.out.println(name));
-```
-The following API classes also include ```getXxxxxStream()``` methods which return a Java 8 Stream:
-```
-GroupApi
-IssuesApi
-NotesApi
-ProjectApi
-RepositoryApi
-TagsApi
-UserApi
+// Stream method
+Stream projectsStream<Project> = gitlabApi.getProjectApi().getProjectsStream();
```

Comment:
duplicated Stream<Project> at the beginning

Reply:
Fixed.